src/hooks/useComments.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { apiUrl } from "../utils/apiURL";
+
+export const useComments = () => {
+  const [comments, setComments] = useState([]);
+  const [cashedComment, setCashedComment] = useState([]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (error) {
+      setComments((curr) => [cashedComment[0], ...curr]);
+      console.log(comments);
```

Comment:
I'm sure this is just here while you're figuring stuff out, but don't leave console.logs in your code when it's deployed!

Reply:
There's a few others too - I would just search your directory for console.logs and remove them.